_includes/comment-form.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<hr size=10>
+<head>
+  <meta charset="utf-8"/>
+  <title>Comment Section</title>
+
+</head>
+<body>
+
+    <div class="form-header header-blue">
+      <h4><i class="fa fa-comments"></i> Neues Kommentar </h4>
+    </div><!-- end .form-header section -->
+
+    <form method="POST"
+          action="https://staticman.intia.de/v2/entry/janseidler/portal.intia.de/main/comments"
+          id="comment2">
+
+      <div class="section">
```

Comment:
Probier das mal aus, damit das richtig formatiert ist mit dem icon (ähnlich sollte auch für die anderen gelten):
<div class="section">
        <label class="field">
          <label class="label">Name</label>
          <div class="control has-icons-left">
            <input type="text" name="fields[name]" id="name" class="input is-primary"
                 placeholder="(Optional) Name eingeben..." value="Anonymer Mensch">
            <span class="icon is-small is-left"><i class="fa fa-user"></i></span>
          </div>
        </label>
      </div><!-- end section -->